apps/web/components/CategoryFilters/Filter.vue

```diff
@@ -2,56 +2,59 @@
   <SfAccordionItem v-if="facet" v-model="open">
     <template #summary>
       <div class="flex justify-between p-2 mb-2">
-        <p class="mb-2 font-medium typography-headline-5">{{ facet.label }}</p>
+        <p class="mb-2 font-medium typography-headline-5">{{ facet.name }}</p>
```

Comment:
We usually use getters here, but facets don't have getters to get the actual data. Should we implement them as we go ? @maxiroellplenty ?